src/main/java/faang/school/godbless/error_prettifier/PrettyError.java

```diff
@@ -0,0 +1,15 @@
+package faang.school.godbless.error_prettifier;
+
+import java.util.function.Supplier;
+
+public class PrettyError{
+    public static <T> T withErrorHandling(Supplier<T> supplier, ExceptionHandler exceptionHandler) {
+        try {
+            return supplier.get();
+        } catch (RuntimeException e) {
+            exceptionHandler.handle(e);
+        }
+//        idk how it can be handled in parametrized function without null
```

Comment:
how about returning 10th line?

Reply:
Aight you could just change the method in the interface to T from void and just return 10th line